data/blog/2024-year-in-review.mdx

```diff
@@ -0,0 +1,545 @@
+desperately lacks, as well as a new paradigm on how to build stuff for the web.
+One that is resistant to censorship and downtime alike; one that has
+pseudonymous identity and lightning-fast payments built in. To say that we are
+bullish would be an understatement; we are exhilarated to see it all come
+together.
+
+But much is still left to be done. Bitcoin, Lightning, and nostr are far from
+perfect, and as these and other protocols are evolving, we will do our best to
+support the projects and builders who are moving the needle. More than that: not
+only do we want to support innovation and improvements, but we first and
+foremost want to make sure that those who are maintaining, reviewing, and
+stress-testing critical infrastructure are taken care of. There are many unsung
+heroes that make all this stuff work, and we are here to support them.
+
+Our generous donors make our support for free and open-source software possible.
+One of our goals for 2025 is to receive multiple recurring commitments to ensure
+that our long-term support programs are well-funded. After all, [our
+mission](/mission) is to maintain a sustainable ecosystem of funding for free
+and open-source projects and contributors. We hope that our track record will
+incentivize those who rely on bitcoin to give to OpenSats periodically, both to
+one of our funds as well as to our operations budget, which we use to
+independently fund our small team. We are one of the few charities that don’t
+take a cut from donations, so if you think we are doing good work and you want
+us to continue to exist, please continue to support us:
+
+<center>
+    <DonateToOperationsButton/>
+</center>
+
+Our hope is to scale our grant-giving efforts considerably going forward, and we
+will share some thoughts on this matter at the beginning of the new year. In the
+best case, we want to see many more organizations like OpenSats, as we believe
+that specialization and a broader support network are paramount as the various
+ecosystems grow and specialize. We don’t have a crystal ball of course, but from
+our perspective, it isn’t hard to imagine that nostr and ecash are here to stay
+and are bound to grow, and it isn’t outlandish to think that open-source
+hardware will gain in importance as well. Bitcoin will remain to be of the
+utmost importance of course, and as it enters the political arena of the world
+stage we expect to see multiple non-profits being created to support open-source
+developers across all kinds of jurisdictions. If you are thinking about creating
+such an organization don’t hesitate to [reach out to us](/contact).
+
+Last but not least, we want to dedicate this final paragraph to all of you.
+Whether you are a donor, grantee, future applicant, or simply an interested
+reader who is interested in bitcoin and freedom tech—the revolution that is
+currently happening is only happening because of you. We are incredibly thankful
```

Comment:
(several uses of the word "incredibly" in this post, perhaps vary with other words)

Reply:
Well spotted! Down to one now. 

Thank you for the in-depth review and the great suggestions, I truly appreciate it 🙏🧡